docs/src/main/asciidoc/_client-guide/authentication-client.adoc

```diff
@@ -292,6 +292,47 @@ The individual match elements are: -
 * <match-domain-name name="..."/>:: Matches where the protocol of the URI is 'domain' and the scheme specific part of the URI is the name specified within this match element.
  <match-abstract-type name="..." authority="..." />:: Matches the abstract type and/or authority against the values specified within this match element.
 
+=== Elytron client default SSL context security provider
+
+You can register the `org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider` security provider when you have runtime dependency on wildfly-elytron-client. This java security provider loads Elytron client configuration and provides an initialized default SSL context. You can make use of this SSL context by calling `SSLContext.getDefault()` if this provider is registered with high enough priority.
```

Comment:
Maybe the first sentence could be reworded to something like:

To make use of the `org.wildfly.security.auth.client.WildFlyElytronClientDefaultSSLContextProvider`, a runtime dependency on `wildfly-elytron-client` is needed.

Reply:
Fixed